crates/next-core/src/env.rs

```diff
@@ -47,6 +47,29 @@ pub async fn load_env(project_path: FileSystemPathVc) -> Result<ProcessEnvVc> {
     Ok(env)
 }
 
+#[turbo_tasks::function]
+pub async fn full_env(
```

Comment:
Can you document how this differs from `env_for_js`

Reply:
Also, where is this used?

Reply:
`full_env` is a bit misleading as name. This is the `filtered_env` for client?

Reply:
This is unused actually, I was looking into different ways to implement this

I wanted to partially get rid of the injected env variables, but maybe I'll just undo that